src/components/GenuineCheck.js

```diff
@@ -134,6 +134,7 @@ class GenuineCheck extends PureComponent<Props> {
           </Trans>
         ),
         icon: genuineCheckIcon,
+        timeout: 30000,
```

Comment:
let's define that in a constant (if not already the case)

Reply:
so wait, there is something i'm not sure to understand here.

the timeout is not supposed to interrupt the step.

the feature is supposed to just be a warning, but the task will continue.

the genuine timeout currently throw 2min after, but here,  it's just to implement a lil' message after 30s 🤔  

if this is what you have implemented (which i don't see it is 🤔  ) i think it should be renamed. 

we basically want ability to "notify" , maybe each second or at a given rate 🤔

Reply:
So it should not be an error, and should not provide "skip" feature?